sparse/unit_test/Test_Sparse_par_ilut.hpp

```diff
@@ -324,4 +319,4 @@ void test_par_ilut() {
 #include <Test_Common_Test_All_Type_Combos.hpp>
 
 #undef KOKKOSKERNELS_EXECUTE_TEST
-#define NO_TEST_COMPLEX
+#undef NO_TEST_COMPLEX
```

Comment:
This was a significant error that could have disabled complex testing in the subsequent test.